ProjectLighthouse/Match/Rooms/RoomHelper.cs

```diff
@@ -184,7 +184,7 @@ public static Room CreateRoom(List<int> users, GameVersion roomVersion, Platform
     }
 
     [SuppressMessage("ReSharper", "InvertIf")]
-    public static void CleanupRooms(int? hostId = null, Room? newRoom = null, Database? database = null)
+    public static Task CleanupRooms(int? hostId = null, Room? newRoom = null, Database? database = null)
```

Comment:
Why are we making this a task? Does this somehow improve performance?

Reply:
It was to get rid of a warning because repeating tasks use task and the function returned void